src/js/lib/proxies.js

```diff
@@ -34,33 +34,6 @@
 
 import { fromQuerystring, getHostName } from './helpers';
 
-/*
```

Comment:
Do you know what this is/why it was created in the first place?

Not that I see any reason to object to removing it, just hard to think about without knowing what it is. :)

Reply:
This was how Yahoo used to show cached pages, they just use Bing now though.